db/migrate/20230210115830_add_qualified_teachers_api_status_and_sent_to_qualified_teachers_api_at_to_participant_outcomes.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddQualifiedTeachersApiStatusAndSentToQualifiedTeachersApiAtToParticipantOutcomes < ActiveRecord::Migration[6.1]
+  def change
+    add_column :participant_outcomes, :qualified_teachers_api_status, :boolean, null: false, default: false
```

Comment:
I wonder if we can find a better name for this, it took me a while to remember what this value was capturing, which was if a participant outcome request to the API was successful and didn't result in an error 🤔 
maybe `qualified_teachers_api_successful`? Let me know your thoughts

Reply:
or `qualified_teachers_api_request_succeeded_at` with a timestamp (which may or may not be useful to know)?

Reply:
I think we were hoping to avoid a timestamp on success because a failure would result in `nil` values, so we couldn't capture if we needed to re-retry this job or not. We were hoping to avoid re-queueing genuine API failures due to for instance the TRN not being available on the TRA side.

The other timestamp column hopefully captures a queue attempt, at the job level. 

So the main data we're trying to capture is:
- Queued to send to TRA
- Response received from TRA successfully or unsuccessfully

Response details/timestamps are captured in the other model hopefully for auditing
Let me know your thoughts

Reply:
It might be good to keep "tra" in the name as that is what we use to refer to this API when we speak generally.

Reply:
Tks guys made the suggested changes... 🔝 😄 👍